### PR TITLE
Tests: add tests for storage

### LINKED ISSUE #223: Test: add tests for `Storage`

### PR DESCRIPTION
**Pull request by jasonqiu212**
## Changes
- Increase code coverage by 1.40%

## Issues
- Resolve #223

### CODE REVIEW

File: src/test/java/swift/storage/JsonAdaptedBridgeTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+import org.junit.jupiter.api.Test;
+
+import swift.commons.exceptions.IllegalValueException;
+
+public class JsonAdaptedBridgeTest {
+    private static final String INVALID_PERSON_ID = "invalid";
+    private static final String INVALID_TASK_ID = "invalid";
+
+    private static final String valid_person = DEFAULT_BRIDGE_1.getPersonId().toString();
```
Comment (yunruu) [inline src/test/java/swift/storage/JsonAdaptedBridgeTest.java:18]: Variable should be capitalized as it is a constant i.e. VALID_PERSON :)

File: src/test/java/swift/storage/JsonAdaptedBridgeTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    @Test
+    public void toModelType_invalidPersonId_throwsIllegalValueException() {
+        JsonAdaptedBridge bridge = new JsonAdaptedBridge(INVALID_PERSON_ID, VALID_TASK_ID);
+        assertThrows(IllegalValueException.class, bridge::toModelType);
+    }
+
+    @Test
+    public void toModelType_nullPersonId_throwsIllegalValueException() {
+        // This test checks for null person id
```
Comment (yunruu) [inline src/test/java/swift/storage/JsonAdaptedBridgeTest.java:35]: Pretty self-explanatory so comment is not really necessary :)

File: src/test/java/swift/storage/JsonAdaptedBridgeTest.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    @Test
+    public void toModelType_nullTaskId_throwsIllegalValueException() {
+        JsonAdaptedBridge bridge = new JsonAdaptedBridge(valid_person, null);
+        String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, UUID.class.getSimpleName());
+        assertThrows(IllegalValueException.class, expectedMessage, bridge::toModelType);
+    }
+
+//    @Test
```
Comment (yunruu) [inline src/test/java/swift/storage/JsonAdaptedBridgeTest.java:54]: Please remove commented out code :)

**Review by yunruu — CHANGES_REQUESTED** [review]
LGTM but just some slight changes to be made 😄

File: src/test/java/swift/storage/JsonAdaptedBridgeTest.java (hunk 1 above)
Comment (jasonqiu212) [reply src/test/java/swift/storage/JsonAdaptedBridgeTest.java:18]: thanks! good catch

**Review by yunruu — APPROVED** [review]
LGTM, thanks for the changes. Will merge it in 👍

**Pull request merged**